File: Java/MathematicalPrograms/SquaresSquareRootsCubesCubeRoot.java
import java.util.Scanner;
import java.lang.Math;
public class SquaresAndCubes {
	public static void main(String[] args) {
     Scanner myObj = new Scanner(System.in);
	 System.out.println("Enter a number");
	 double numper1 = myObj.nextInt();
	 System.out.println("Enter another number");
	 double numper2 = myObj.nextInt();
	 System.out.println("Enter a number");
	 double numper3 = myObj.nextInt();
	 System.out.println("Enter a number");
	 double numper4 = myObj.nextInt();
	 double square1 = Math.pow(numper1, 2);
	 double squareroot2 = Math.sqrt(numper2);
	 double cube3 = Math.pow(numper3, 3);
	 double cuberoot = Math.cbrt(numper4);
	 System.out.println("The square of number " + numper1 + " is " + square1 + ". The square root of " + numper2 + " is " + squareroot2 + ". The cube of number " + numper3 + " is " + cube3 + ". The cube root of " + numper4 + " is " + cuberoot);
	}
}

File: C++/MathematicalPrograms/SquaresSquareRootsCubesCubeRoot.cpp
#include <iostream>
#include <string>
#include <math.h>

using namespace std;

int main() {
	double num1, num2, num3, num4;
	double square1, squareroot2, cube3, cuberoot4;
	cout << "Enter a number";
	cin >> num1;
	cout << "Enter another number";
	cin >> num2;
	cout << "Enter another number";
	cin >> num3;
	cout << "Enter another number";
	cin >> num4;
	square1 = pow(num1, 2);
	squareroot2 = sqrt(num2);
	cube3 = pow(num3, 3);
	cuberoot4 = cbrt(num4);
	cout << "The square of number "<< num1 << " is " << square1 << ". The square root of " << num2 << " is " << squareroot2 << ". The cube of number " << num3 << " is " << cube3 << ". The cube root of " << num4 << " is " << cuberoot4;
}